sdks/csharp/twin_operations_other.cs

```diff
@@ -80,3 +80,16 @@ class MyCustomDigitalTwinMetadata
 entity-02: "tags": { "purple": "", "size": "small" }
 entity-03: "tags": { "red": "", "size": "small" }
 // </TagPropertiesValue>
+
+// ------------------ SET TAG PROPERTY VALUES: CSHARP ---------------------
+// <TagPropertiesCsharp>
+BasicDigitalTwin twin = new BasicDigitalTwin();
```

Comment:
I think the variable `twinData` is intended to be the one instantiated as `twin`.

Also, here are some inline ways to do the initialization.

```csharp
var twin = new BasicDigitalTwin
{
    Metadata = { ModelId = "dtmi:example:Room;1",
    Contents =
    {
        { "Temperature", 75 },
    },
};
```

Reply:
Thank you @drwill-ms, twinData was a bad case of copy and paste from my own sample code. I also adjusted the initialization per your recommendation. Thank you!